### DISCUSSION TITLE
Icon Request: fa-licenseplate

### DISCUSSION THREAD

**Discussion by jrdad1** [FortAwesome/Font-Awesome#21149] [Icon Requests]
A license plate that is on a vehicle such as an automobile
I will be using the icon in a tab but can be used in any transportation system that is related to cars, taxi, and other vehicles.

![license plate](https://cloud.githubusercontent.com/assets/11945723/7140071/87fb3ea6-e297-11e4-84bc-b2d813a9826c.gif)

**Comment by Izopi4a** [comment]
:+1:

**Comment by tagliala** [comment]
hmmm... Is there a better example?

**Comment by jrdad1** [comment]
![license plate](https://cloud.githubusercontent.com/assets/11945723/7179738/ce528720-e408-11e4-8f40-cdb76d885799.gif)

**Comment by lorinmetzger** [comment]
:+1:

**Comment by tagliala** [comment]
this is hardly suitable for 14px icon size

**Comment by jrdad1** [comment]
It was only an example. You can make it simpler.

**Comment by juques** [comment]
+1

**Comment by jrdad1** [comment]
![lic_plate](https://cloud.githubusercontent.com/assets/11945723/12427486/9e8b247c-bead-11e5-84eb-f5d023814da7.gif)

**Comment by jeroen-corteville** [comment]
+1

**Comment by DoubleDot** [comment]
+1

**Comment by portal7** [comment]
+1

**Comment by dedo1911** [comment]
+1

**Comment by darma-avalos** [comment]
+1

**Comment by richardkeller411** [comment]
+1

**Comment by baris-unver** [comment]
+1

**Comment by vasc0x** [comment]
+1

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂

**Comment by xdarkfirestorm** [comment]
+1

**Comment by IntissarJ** [comment]
+1

**Comment by wesleydarnell** [comment]
+1
Food for thought icon:
<img width="169" height="100" alt="license_plate" src="https://github.com/user-attachments/assets/727711c1-cf9c-47af-96aa-97405c5c6c07" />
<img width="27" height="16" alt="license_plate" src="https://github.com/user-attachments/assets/727711c1-cf9c-47af-96aa-97405c5c6c07" />

**Comment by NicoGomezG** [comment]
+1
